src/ray/gcs/gcs_server/gcs_server.h

```diff
@@ -110,6 +110,11 @@ class GcsServer {
   static constexpr char kInMemoryStorage[] = "memory";
   static constexpr char kRedisStorage[] = "redis";
 
+  GcsResourceManager &GetGcsResourceManager() {
```

Comment:
Maybe instead of exposing this, add a `UpdateNodeResourceUsageForTesting` method?